package.json

```diff
@@ -54,6 +54,7 @@
     "@nestjs/core": "^10.0.5",
     "@nestjs/schematics": "^10.0.1",
     "@nestjs/testing": "^10.0.5",
+    "@swc/core": "^1.3.78",
```

Comment:
Is it necessary `swc` here? https://tsup.egoist.dev/#install

Reply:
I think so https://tsup.egoist.dev/#error-no-matching-export-in-xxx-ts-for-import-xxx